### PR TITLE
Remove rktnetes labels and configs

### PR DESCRIPTION
**Pull request by yujuhong**
rktnetes has been deprecated and the code is being removed.

### CODE REVIEW

File: label_sync/labels.yaml [hunk 1]
```diff
@@ -158,8 +158,6 @@ labels:
     name: sig/openstack
   - color: d2b48c
     name: sig/release
-  - color: d2b48c
-    name: sig/rktnetes
```
[the remark this answers is not in the document]
Comment (yujuhong) [reply label_sync/labels.yaml]: Done and thanks!

**Comment by BenTheElder** [comment]
needs rebase

**Pull request merged**